example/advanced/server-flex/advanced_server_flex.cpp

```diff
@@ -106,18 +105,14 @@ path_cat(
     return result;
 }
 
-// This function produces an HTTP response for the given
-// request. The type of the response object depends on the
-// contents of the request, so the interface requires the
-// caller to pass a generic lambda for receiving the response.
-template<
-    class Body, class Allocator,
-    class Send>
-void
+// This function produces an HTTP response for the given request.
```

Comment:
For functions which return meaningful values I start the brief with "Return...". For example:

"Return a response for the given HTTP request."